sdk/keyvault/azure-keyvault-keys/tests/test_examples_crypto.py

```diff
@@ -18,6 +18,18 @@
 )
 
 
+def test_create_client_from_jwk():
+    # [START from_jwk]
+    # create a CryptographyClient using a JsonWebKey instance
+    key = JsonWebKey(kty="RSA")
```

Comment:
If someone copies this, it will definitely not work 😜 

Which points to an interesting user experience problem. If I give CryptographyClient a garbage JWK, it won't complain until I ask it to perform an operation (and then it just says the key doesn't support that operation). This delayed error was the best we could do before, because it wasn't feasible to determine at construction what would be possible with a given key or key ID. That's no longer true in local-only mode though. The client could raise ValueError when given a key that it can't get a local provider for, for example.